### PR TITLE
Updated deprecated type to @type

### PR DESCRIPTION
**Pull request by AmiSMB**

### CODE REVIEW

File: pkg/deb/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+google-fluentd-catch-all-config (0.6) unstable; urgency=medium
+
+  * Updated deprecated type to @type.
```
Comment (igorpeshansky) [inline pkg/deb/debian/changelog:3]: Let's use present tense, for consistency with other comments.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
I've commented on the content, but unfortunately, we need a CLA before we can proceed.

**Comment by JacobHayes** [comment]
@AmiSMB any update on signing the CLA? Otherwise, @igorpeshansky, could I re-submit this PR with a signed CLA?

**Comment by AmiSMB** [comment]
I signed it!

**Comment by igorpeshansky** [comment]
Great, let's address the pending review comments.

**Comment by igorpeshansky** [comment]
Thanks for the contribution, but I believe this has been superseded by #15.

**Pull request closed without merging**